packages/perseus/src/widgets/interactive-graphs/graphs/components/angle-indicators.tsx

```diff
@@ -0,0 +1,428 @@
+    const angle = (startAngle + 360 - endAngle) % 360;
+    const halfAngle = (endAngle + angle / 2) % 360;
+    const angleLabel = parseFloat(angle.toFixed(0)); // Only want to show whole angles
+
+    // Check if the angle is reflexive and if we should allow it
+    const isReflexive = angle > 180;
+
+    // Break out the necessary points for the arc calculation
+    const [centerX, centerY] = vertex;
+    const [point1, point2] = clockwiseCoords;
+    const [startX, startY] = point1;
+    const [endX, endY] = point2;
+    const a = vec.dist(vertex, point1);
+    const b = vec.dist(vertex, point2);
+
+    // Set the radius of the arc
+    const radius = 2;
+
+    // Calculate the end points of the arc
+    const y1 = centerY + ((startY - centerY) / a) * radius;
+    const x2 = centerX + ((endX - centerX) / b) * radius;
+    const x1 = centerX + ((startX - centerX) / a) * radius;
+    const y2 = centerY + ((endY - centerY) / b) * radius;
+
+    // Used to calculate the inside of the angle and to create square
+    const [x3, y3] = vec.add(
+        vertex,
+        vec.add(vec.sub([x1, y1], vertex), vec.sub([x2, y2], vertex)),
+    );
+
+    // Determine whether the arc should be drawn outside the angle
+    const isOutside = shouldDrawArcOutside([x3, y3], vertex, range, [
+        [vertex, point1],
+        [vertex, point2],
+    ]);
+
+    // Determine the flags for the arc
+    // If the angle is outside OR reflexive, we want to draw a large arc
+    // If the angle is outside AND reflexive, we want to sweep the arc
+    const largeArcFlag = isOutside || isReflexive ? 1 : 0;
+    const sweepFlag = isOutside && isReflexive ? 1 : 0;
+
+    // Create the SVG path for the arc
+    const arc = `M ${x1} ${y1} A ${radius} ${radius} 0 ${largeArcFlag} ${sweepFlag} ${x2} ${y2}`;
+
+    // Calculate the text position based on the angle and whether we allow reflex angles
+    // Let's try the angle bisector method to find the midpoint of the arc
+
+    const [textX, textY] = calculateBisectorPoint(
+        point1,
+        point2,
+        vertex,
+        isReflexive,
+    );
+
+    return (
+        <>
+            <defs>
+                <filter
+                    id="background"
+                    x="-5%"
+                    width="110%"
+                    y="0%"
+                    height="100%"
+                >
+                    <feFlood floodColor="#FFF" floodOpacity="0.5" />
+                    <feComposite operator="over" in="SourceGraphic" />
+                </filter>
+            </defs>
+
+            {!isReflexive && isRightAngle(angle) ? (
+                <RightAngleSquare
+                    start={[x1, y1]}
+                    vertex={[x2, y2]}
+                    end={[x3, y3]}
+                />
+            ) : (
+                <MafsCssTransformWrapper>
+                    <path
+                        d={arc}
+                        strokeWidth="2000"
```

Comment:
Fix stroke width. This was a test that discovered the stroke width is doing nothing.